packages/notebook/src/browser/contributions/notebook-outline-contribution.ts

```diff
@@ -21,18 +21,16 @@ import { OutlineViewService } from '@theia/outline-view/lib/browser/outline-view
 import { NotebookModel } from '../view-model/notebook-model';
 import { OutlineSymbolInformationNode } from '@theia/outline-view/lib/browser/outline-view-widget';
 import { NotebookEditorWidget } from '../notebook-editor-widget';
-import { NotebookCellModel } from '../view-model/notebook-cell-model';
 import { DisposableCollection, URI } from '@theia/core';
 import { CellKind, CellUri } from '../../common';
 import { NotebookService } from '../service/notebook-service';
 export interface NotebookCellOutlineNode extends OutlineSymbolInformationNode {
-    notebookCell: NotebookCellModel;
-    uri: URI;
+    uri: string;
```

Comment:
Suggestion: Independent of the comment above, we should continue using `URI` here. The `OutlineViewWidget` inflates nodes back using the `URI` object. See [here](https://github.com/eclipse-theia/theia/blob/9c9456b8f7d32f9788d60f83377f109f1743aaf5/packages/outline-view/src/browser/outline-view-widget.tsx#L208). This should make the rest of the code better to maintain as well.

Reply:
ah i didn't know that. I saw some error because of a path element so i assumed it was because of the uri as well. Should be better now